config/application.rb

```diff
@@ -25,5 +25,6 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.paths["log"] = ENV["LOG_PATH"] if ENV["LOG_PATH"]
+    GDS::SSO::Config.api_only = true
```

Comment:
This would be more consistent in the initializer as per: https://github.com/alphagov/support-api/blob/1281113d26c5edff8af43e1a6b6e410bd574effd/config/initializers/gds-sso.rb#L7

Reply:
fixed